zhaquirks/tuya/__init__.py

```diff
@@ -1507,6 +1507,9 @@ class TuyaNewManufCluster(CustomCluster):
     cluster_id: t.uint16_t = TUYA_CLUSTER_ID
     ep_attribute: str = "tuya_manufacturer"
 
+    class AttributeDefs(CustomCluster.AttributeDefs):
```

Comment:
Ok, I guess `CustomCluster.AttributeDefs` works.
Just to stay consistent with other quirks, I'd also use `BaseAttributeDefs` here.
Yeah, it's basically the same, but.. 😄 

```suggestion
    class AttributeDefs(BaseAttributeDefs):
```
(might also need to import `BaseAttributeDefs`)